main/main.c

```diff
@@ -512,35 +505,17 @@ void app_main(void)
     }
     ESP_ERROR_CHECK(ret);
 
-    // if lyra or custom
-    //board_handle = audio_board_init();
-    //audio_hal_ctrl_codec(board_handle->audio_hal, AUDIO_HAL_CODEC_MODE_BOTH, AUDIO_HAL_CTRL_START);
-    //i2s_mclk_gpio_select(0,0);
+	ESP_LOGI(TAG, "Start codec chip");
+    board_handle = audio_board_init();
+    audio_hal_ctrl_codec(
+		board_handle->audio_hal, AUDIO_HAL_CODEC_MODE_BOTH, AUDIO_HAL_CTRL_START);
+    i2s_mclk_gpio_select(0, 0);
 
-    //audio_hal_set_volume(board_handle->audio_hal,5);
-
-    setup_ma120();
-    //ma120_setup_audio(0x20);
-    //setup_ma120x0();
-
-    uint8_t rxbuf[12];
-    ma_read(0x20,2,0x0003, rxbuf,3);
-    printf("\nVolume : 0x%02x 0x%02x 0x%02x\n",rxbuf[0], rxbuf[1], rxbuf[2] );
-    ma_write_byte(0x20,2,0x608, 51);
+    // ma_write_byte(0x20,2,0x608, 51); // XXX what's this for?
```

Comment:
Note the XXX also here, no idea why/what this is for.

Reply:
Early prototype hack - internal RC oscillator trimming. Not needed anymore.